packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx

```diff
@@ -68,9 +77,10 @@ const Step = ({ phase, currentPhase }: StepProps) => {
 
 interface PhaseProgress {
     currentPhase: RoundPhase;
+    phaseDeadline: number | string;
```

Comment:
Can this really be a string? It seems the type can be narrowed down in `CoinjoinSession` interface and also here to just `number`.

Reply:
I briefly discussed it with Szymon and it was decided to keep it like that :)